content/dart/concepts/list/terms/addAll/addAll.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.addAll()'
+Description: 'Adds multiple objects from a list.'
+Subjects:
+  - 'Computer Science'
+  - 'Mobile Development'
+  - 'Code Foundations'
+Tags:
+  - 'Dart'
+  - 'Arithmetic'
+  - 'Lists'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+The **`.addAll()`** method in Dart which allows you to append multiple elements from one list to another list. This method is useful when you want to combine the contents of multiple lists into a single list. This method does not create a new list rather modifies the original list.
+
+## Syntax
+
+```pseudo
+list1.addAll(list2);
+```
+

```

Comment:
Describe the list1 and list2 parameters just below the syntax block. It should be in this format - 
```
- `paramter`: it's description.
```